include/openssl/cms.h.in

```diff
@@ -96,7 +96,8 @@ CMS_ContentInfo *CMS_ContentInfo_new_ex(OSSL_LIB_CTX *libctx, const char *propq)
 const ASN1_OBJECT *CMS_get0_type(const CMS_ContentInfo *cms);
 
 BIO *CMS_dataInit(CMS_ContentInfo *cms, BIO *icont);
-int CMS_dataFinal(CMS_ContentInfo *cms, BIO *bio);
+int CMS_dataFinal(CMS_ContentInfo *cms, BIO *bio,
```

Comment:
Oups ! you cannot change signatures of public API methods.

Reply:
Ok, I will create a CMS_dataFinalDigest. (Didn't know this is public since I didn't see it documented i any pod file.)

Reply:
Every file in `include` root folder is part of Public API.
This one is the file you call when you write :
`#include <openssl/cms.h>`

Reply:
> Every file in `include` root folder is part of Public API.

Incorrect. Only files in `include/openssl` are the public API.